cristin-commons/src/main/java/no/unit/nva/cristin/common/model/SearchResponse.java

```diff
@@ -28,7 +28,7 @@
     JsonPropertyNames.CONTEXT, JsonPropertyNames.ID, JsonPropertyNames.SIZE, JsonPropertyNames.SEARCH_STRING,
     JsonPropertyNames.PROCESSING_TIME, JsonPropertyNames.FIRST_RECORD, JsonPropertyNames.NEXT_RESULTS,
     JsonPropertyNames.PREVIOUS_RESULTS, JsonPropertyNames.HITS})
-public class SearchResponse {
+public class SearchResponse<E> {
```

Comment:
E? Element (in Collection), or T (Type)?

Reply:
SearchResponse has Elements in collection?